llvm/test/CodeGen/NVPTX/atomics-sm70.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc < %s -march=nvptx -mcpu=sm_70 | FileCheck %s
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_70 | FileCheck %s
```

Comment:
Do these tests still work? If they do, then we have a bug somewhere because -mcpu=sm70 would set PTX to 6.0 and that should make `atom.add.noftz.f16` unavailable.

At the very least the test should have set ptx version via `-mattr=+ptx63`

Speaking of PTX version, you may want to have test cases for when the atomic instruction is not available.

Reply:
You are right, and it also shows the failure in presubmit testing. Sorry for missing that. And thanks for the tip with setting -mattr=+ptx63. I also added tests with -mattr=+ptx62 to show that it is not using atom.add.noftz.f16 in that case.